lib/rocket_chat/messages/im.rb

```diff
@@ -0,0 +1,48 @@
+module RocketChat
+  module Messages
+    #
+    # Rocket.Chat Direct messages
+    #
+    class Im
+      #
+      # @param [Session] session Session
+      #
+      def initialize(session)
+        @session = session
+      end
+
+      #
+      # im.create REST API
+      # @param [String] username Rocket.Chat username
+      # @return [RocketChat::Room]
+      # @raise [HTTPError, StatusError]
+      #
+      def create(username: nil)
```

Comment:
given `username` is required, why specify a default? Should be `create(username:)`